cartographer_ros/cartographer_ros/assets_writer.cc

```diff
@@ -241,4 +244,40 @@ void RunAssetsWriterPipeline(const std::string& pose_graph_filename,
            carto::io::PointsProcessor::FlushResult::kRestartStream);
 }
 
+}  // namespace
+
+void RunAssetsWriterPipeline(const std::string& pose_graph_filename,
+                             const std::vector<std::string>& bag_filenames,
+                             const std::string& configuration_directory,
+                             const std::string& configuration_basename,
+                             const std::string& urdf_filename,
+                             const std::string& output_file_prefix,
+                             const bool use_bag_transforms) {
+  carto::mapping::proto::PoseGraph pose_graph_proto;
+  carto::mapping::proto::AllTrajectoryBuilderOptions
+      all_trajectory_builder_options;
+  std::tie(pose_graph_proto, all_trajectory_builder_options) =
+      CreatePoseGraph(pose_graph_filename, bag_filenames.size());
```

Comment:
Naming: `LoadPoseGraph` since this is reading it from a file, not creating a new one?

Reply:
Agree, LoadPoseGraph is more precise.